.idea/comp-loinc.iml

```diff
@@ -3,6 +3,7 @@
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/comp_loinc" isTestSource="false" />
```

Comment:
## `.idea/` files shouldn't be committed
It looks like this folder is in the `.gitignore`, but somehow some of these files slipped in.

This modification is just for my personal use, but it also useful. It marks `src/comp_loinc/` as a source directory, which helps/helped remove some import errors, and will also help it run from different relative directories from PyCharm since it can now resolve those imports.

Most of the Python projects I see don't use `src/`. I don't prefer it, but it's not wrong and I don't entirely dislike it either.

In any case, I created a low-priority issue to remove these and similar files:
- #60